certified-connectors/DocuSignDemo/apiDefinition.swagger.json

```diff
@@ -630,6 +630,14 @@
             "x-ms-visibility": "important",
             "type": "string"
           },
+          { 
```

Comment:
this param seems to be like one you added while debugging. did you forget to remove ?
if not, it needs to be renamed to reflect its purpose.